neoarch/managers/docker_manager.py

```
@@ -550,20 +554,181 @@ def run_thread():
+            }
+            QWidget#dockerImageCard:hover {
+                background-color: rgba(22, 23, 26, 0.7);
+                border-color: rgba(255, 255, 255, 0.08);
+            }
+        """)
+        return card
+
+    def _stop_containers_for_image(self, image):
+        """Stop all containers using the given image."""
+        try:
+            result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)